exporter/prometheusexporter/collector.go

```diff
@@ -109,11 +109,23 @@ func (c *collector) getMetricMetadata(metric pmetric.Metric, attributes pcommon.
 	keys := make([]string, 0, attributes.Len()+2) // +2 for job and instance labels.
 	values := make([]string, 0, attributes.Len()+2)
 
-	attributes.Range(func(k string, v pcommon.Value) bool {
-		keys = append(keys, prometheustranslator.NormalizeLabel(k))
-		values = append(values, v.AsString())
-		return true
-	})
+	var metricName string
+
+	if allowUTF8FeatureGate.IsEnabled() {
+		attributes.Range(func(k string, v pcommon.Value) bool {
+			keys = append(keys, k)
+			values = append(values, v.AsString())
+			return true
+		})
+		metricName = metric.Name()
```

Comment:
This needs to change. The config file have the option `add_metric_suffixes`, so we still need to honor it